freqtrade/strategy/default_strategy.py

```diff
@@ -199,7 +199,7 @@ def populate_indicators(self, dataframe: DataFrame, metadata: dict) -> DataFrame
 
         return dataframe
 
-    def populate_buy_trend(self, dataframe: DataFrame, metadata: dict) -> DataFrame:
+    def populate_buy_trend(self, dataframe: DataFrame, metadata: dict = {}) -> DataFrame:
```

Comment:
can you help me understand what an empty dict will do for us here? (same for the other 2 function headers)

if the function accepts to parameters, the pair is passed in - if not, the "old" function-header needs to be called.

If this dict where empty (so wrongly called) every Strategy relying on the metadata dict will fall flat on it's face as it'll error out with "key not found".

Reply:
This was again on the bslap branch. my guess is creslin or berlin wanted to pass winrate parameters to strategy but with current code it is not needed. will be removed

Reply:
yeh i have that in another stream  - cross contamination.